src/WinUIEx/WindowEx.cs

```diff
@@ -438,6 +440,18 @@ public double MinHeight
             }
         }
 
+        private bool _IsForcedTowardsCenter = false;
+        /// <summary>
+        /// A bool to set whether offsets are forced away from the specified edge 
+        /// </summary>
+        public bool IsForcedTowardsCenter
+        {
+            get => _IsForcedTowardsCenter;
+            set => _IsForcedTowardsCenter = value;
+        }
```

Comment:
I would remove this - it doesn't really make sense as a property on the window, when it only relates to specific operations, and not behavior.
I don't see you using this anywhere either - only in the sample itself.

Reply:
Done, I've moved it to mainwindow